scout/parse/pedqc.py

```diff
@@ -147,3 +149,52 @@ def parse_peddy_sex_check(lines):
             sex_check.append(ind_info)
 
     return sex_check
+
+
+def parse_somalier_pairs(lines: List[str]) -> List[dict]:
+    """Parse a Somalier pairs tsv file"""
+    somalier_pairs = []
+    for i, line in enumerate(lines):
```

Comment:
This code is repeated in the functions following this one. Consider using a more general function that parses somalier files that can be called in each of the 3 functions instead

Reply:
Yep, and a not dissimilar one in the Peddy etc. Its almost like its a TSV/CSV parser.. 😁 I'll see what I can do locally with small effort. 😉